sdk/personalizer/Azure.AI.Personalizer/api/Azure.AI.Personalizer.netstandard2.0.cs

```diff
@@ -257,12 +329,13 @@ public partial class PersonalizerRankableAction
         public PersonalizerRankableAction(string id, System.Collections.Generic.IEnumerable<object> features) { }
         public System.Collections.Generic.IList<object> Features { get { throw null; } }
         public string Id { get { throw null; } }
+        public int Index { get { throw null; } set { } }
     }
     public partial class PersonalizerRankedAction
     {
         internal PersonalizerRankedAction() { }
-        public string Id { get { throw null; } }
-        public float? Probability { get { throw null; } }
+        public string Id { get { throw null; } set { } }
```

Comment:
Should these be settable via ctors and stay read-only, or does it make sense to keep them modifiable after the object is created?

Reply:
Removed the setter